package.json

```diff
@@ -59,17 +59,47 @@
       "order": 1
     },
     "printWidth": {
-      "title": "Print Width",
-      "description": "Fit code within this line limit",
-      "type": "integer",
-      "default": 80,
+      "type": "object",
+      "title": "Print Width Settings",
+      "description": "Code fit settings",
```

Comment:
Atom doesn't seem to use the description for objects so we should probably get rid of this line for clarity